ibis/expr/types/geospatial.py

```diff
@@ -379,6 +854,34 @@ def union(self, right: GeoSpatialValue) -> GeoSpatialValue:
         -------
         GeoSpatialValue
             Union of geometries
+
+        Examples
+        --------
+        >>> import ibis
+        >>> ibis.options.interactive = True
+        >>> import shapely
+        >>> t = ibis.examples.zones.fetch()
+        Penn station zone centroid
```

Comment:
You have to put a newline on either side of the prose, that's why my suggestion had them :)

Otherwise the lines are interpreted as output from the previous repl line.